### DISCUSSION TITLE
Episode 179: CheckGrade, ACCU & CppNorth

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#71] [Episode]
Episode Link: https://adspthepodcast.com/2024/04/26/Episode-179.html

Feel free to write comments or ask questions 🙂

**Comment by CharString** [comment]
Couldn't help but notice the effect our 10x developer has on his support units: the main support doesn't count herself as one of the 3 women in a conference bar. And I knew how the other support took a hit from Nora virus, whilst our 10x developer had brain cycles to burn on grave issues like what walnut furniture won't fit his bigger appartment.

Also is there some Patreon I can join to get the 10x'ed audio of the itinerary of his travels to the middle of nowhere of Ireland? I find it hard to believe he didn't spent at least half an hour on the deals he found, and it's a shame that ended up on the cutting room floor.

**Reply by codereport** [reply]
We don't have a Patreon but maybe we will start one.

**Comment by kamrann** [comment]
I don't think Conor understands the usual connotation of the phrase "fancy math". The 'fancy' aspect is just that there is math.

Also more from Bryce's mum please.
